src/main/index.js

```diff
@@ -67,7 +67,9 @@ app.on('ready', () => {
     }
 })
 
-app.on('window-all-closed', () => {})
+app.on('window-all-closed', () => {
```

Comment:
These changes don't belong to the key navigation. New PR should be open with a visual change. You can leave it now, though.